projects/samples/devices/worlds/vacuum_gripper.wbt

```diff
@@ -0,0 +1,285 @@
+                maxPosition 1.5
+              }
+            ]
+            endPoint Solid {
+              translation 0 0 0.33
+              rotation 0 0 1 1.57078
+              children [
+                DEF ROLL_HINGE_JOINT HingeJoint {
+                  jointParameters HingeJointParameters {
+                    position -1.4131
+                    axis 0 -1 0
+                    anchor 0 -0.3 0
+                  }
+                  device [
+                    RotationalMotor {
+                      name "roll motor"
+                      controlPID 1.5 0 0
+                    }
+                  ]
+                  endPoint Solid {
+                    translation 0 -0.3 0
+                    children [
+                      SliderJoint {
+                        jointParameters JointParameters {
+                          axis 0 -1 0
+                        }
+                        device [
+                          LinearMotor {
+                            name "gripper motor"
+                            acceleration 1
+                            maxVelocity 0.1
+                          }
+                        ]
+                        endPoint VacuumGripper {
+                          rotation 0.5167457898427649 0.6053820811185048 0.6053811398948641 4.095499540775012
+                          children [
+                            Pose {
+                              translation 0 0 0.01
+                              children [
+                                Shape {
+                                  appearance CorrodedMetal {
+                                  }
+                                  geometry Cylinder {
+                                    height 0.04
+                                    radius 0.015
+                                  }
+                                }
+                              ]
+                            }
+                            InertialUnit {
+                              translation 2.59116e-08 2.59116e-08 -0.03
+                              rotation 0.577349935856137 -0.5773509358560258 -0.577349935856137 -2.094395
+                            }
+                            DEF ROLL_GROUP Group {
+                              children [
+                                Pose {
+                                  translation 0 0 -0.02
+                                  rotation 0 0 1 1.5708
+                                  children [
+                                    Shape {
+                                      appearance DEF PLASTIC_APPEARANCE PBRAppearance {
+                                        baseColor 1 0.666667 0
+                                        roughness 1
+                                        metalness 0
+                                      }
+                                      geometry Cylinder {
+                                        height 0.02
+                                        radius 0.04
```

Comment:
```suggestion
                                        radius 0.042
```
Just to avoid this not very nice thing:
![vacuum_gripper](https://github.com/cyberbotics/webots/assets/1264964/1b241c6c-663a-419f-8dac-ac3c23d4863a)